sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -302,7 +304,7 @@ private void setArchitectures(final @NotNull Device device) {
 
   // we can get some inspiration here
   // https://github.com/flutter/plugins/blob/master/packages/device_info/android/src/main/java/io/flutter/plugins/deviceinfo/DeviceInfoPlugin.java
-  private @NotNull Device getDevice() {
+  private @NotNull Device getDevice(final boolean applyScopeData) {
```

Comment:
`l`: Maybe we can find a better name for `applyScopeData`. We use it to control whether we add free and low memory. The name could reflect that.

Reply:
the naming cant be related to free memory, because the same flag would be used to every nonstatic data that cant be applied after the restart, so it's pretty much how we call on every integration that has such check, `applyScopeData`